code/modules/projectiles/updated_projectiles/guns/flamer.dm

```diff
@@ -426,7 +427,7 @@
 		last_use = world.time
 		return
 	if(user.mind?.cm_skills && user.mind.cm_skills.spec_weapons < 0)
-		if(!do_after(user, 10, TRUE, 5, BUSY_ICON_HOSTILE))
```

Comment:
Is the removal of the icon intended here?

Reply:
idem, I guess.